src/lib/editor-routes.js

```diff
@@ -0,0 +1,20 @@
+import { Route, Redirect } from 'react-router';
+import React from 'react';
+import AppController from 'components/editor/AppController';
+import ArticleListController from 'components/editor/ArticleListController';
+import ArticleController from 'components/editor/ArticleController';
+import AuthorListController from 'components/editor/AuthorListController';
+import AuthorController from 'components/editor/AuthorController';
+
+export default (
+  <Route path="/" component={AppController}>
+    <Route path="articles/page/:page/" component={ArticleListController}>
+      <Route path="article/slug/:slug/" component={ArticleController} />
+      <Redirect to="article/slug/:slug" from="article/slug/:slug/" />
```

Comment:
It's more correct that resources do not end in a `/`. The origin of paths was that it would actually access the public web folder on the server computer, so if you ended the path in `/` it meant you ended on a directory and it should show some kind of "list" and when you ended on a path without `/` it means you are at a specific resource.